cmd/kubeadm/app/master/apiclient.go

```diff
@@ -271,9 +269,11 @@ func createDummyDeployment(client *clientset.Clientset) {
 		return true, nil
 	})
 
-	fmt.Println("<master/apiclient> test deployment succeeded")
+	fmt.Println("[apiclient] Test deployment succeeded")
 
-	if err := client.Extensions().Deployments(api.NamespaceSystem).Delete("dummy", &v1.DeleteOptions{}); err != nil {
```

Comment:
It does, tested it now.
Gonna revert this and wait for #38330, but hopefully it will merge very soon